### PR TITLE
Mapped all top-level classes

### PR DESCRIPTION
**Pull request by liach**
No more obfuscated top-level class

Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

**Comment by liach** [comment]
On a side note, these are inner classes yet to map:
```
net/minecraft/datafixers/fixes/LeavesFix$class_1192
net/minecraft/structure/NetherFortressGenerator$class_3404
net/minecraft/structure/OceanMonumentGenerator$class_3367
net/minecraft/structure/OceanMonumentGenerator$class_3368
net/minecraft/structure/OceanMonumentGenerator$class_3369
net/minecraft/structure/OceanMonumentGenerator$class_3370
net/minecraft/structure/OceanMonumentGenerator$class_3371
net/minecraft/structure/OceanMonumentGenerator$class_3372
net/minecraft/structure/OceanMonumentGenerator$class_3373
net/minecraft/structure/OceanMonumentGenerator$class_3375
net/minecraft/structure/OceanMonumentGenerator$class_3388
net/minecraft/structure/StrongholdGenerator$class_3427
net/minecraft/structure/StrongholdGenerator$class_3466
net/minecraft/structure/Structure$class_3500
net/minecraft/structure/WoodlandMansionGenerator$class_3478
```

File: mappings/net/minecraft/client/render/model/json/ItemModelGenerator.mapping [hunk 1]
```diff
@@ -1,5 +1,42 @@
...
+		FIELD field_4274 min I
+		METHOD method_3483 expand (I)V
+		METHOD method_3484 getSide ()Lnet/minecraft/class_801$class_803;
+		METHOD method_3485 getMax ()I
+		METHOD method_3486 getLevel ()I
+		METHOD method_3487 getMin ()I
+	CLASS class_803 Side
+		FIELD field_4276 direction Lnet/minecraft/class_2350;
+		FIELD field_4279 yOffset I
+		FIELD field_4280 xOffset I
```
Comment (Runemoro) [inline mappings/net/minecraft/client/render/model/json/ItemModelGenerator.mapping]: `offsetX`, `offsetY`

File: mappings/net/minecraft/client/render/model/json/ItemModelGenerator.mapping [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+		METHOD method_3488 getDirection ()Lnet/minecraft/class_2350;
+		METHOD method_3489 getYOffset ()I
+		METHOD method_3490 getXOffset ()I
```
Comment (Runemoro) [inline mappings/net/minecraft/client/render/model/json/ItemModelGenerator.mapping]: Here too.

File: mappings/net/minecraft/client/render/model/json/ItemModelGenerator.mapping [hunk 3]
```diff
@@ -1,5 +1,42 @@
...
+		METHOD method_3491 isVertical ()Z
 	FIELD field_4270 LAYERS Ljava/util/List;
+	METHOD method_3477 isPixelTransparent (Lnet/minecraft/class_1058;IIIII)Z
+		ARG 1 sprite
+		ARG 2 frame
+		ARG 3 x
+		ARG 4 y
+		ARG 5 xSize
+		ARG 6 ySize
```
Comment (Runemoro) [inline mappings/net/minecraft/client/render/model/json/ItemModelGenerator.mapping]: `sizeX`, `sizeY`

File: mappings/net/minecraft/entity/passive/PufferfishEntity.mapping [hunk 4]
```diff
@@ -1,8 +1,11 @@
 CLASS net/minecraft/class_1454 net/minecraft/entity/passive/PufferfishEntity
-	CLASS class_1455
+	CLASS class_1455 BlowUpGoal
```
Comment (Runemoro) [inline mappings/net/minecraft/entity/passive/PufferfishEntity.mapping]: I'd call it `InflateGoal`... "blow up" means to explode...

**Review by Runemoro — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**